plugins/webp-uploads/tests/test-picture-element.php

```diff
@@ -264,6 +296,110 @@ function ( string $content ): string {
 		);
 	}
 
+	/**
+	 * Test that the picture element source tag srcset has the same image sizes as the img tag srcset.
+	 *
+	 * @dataProvider data_provider_test_picture_element_source_tag_srcset_has_same_image_sizes_as_img_tag_srcset
+	 * @covers ::webp_uploads_wrap_image_in_picture
+	 *
+	 * @param Closure|null       $set_up Set up the test.
+	 * @param array<string>|null $custom_sizes Custom image sizes.
+	 */
+	public function test_picture_element_source_tag_srcset_has_same_image_sizes_as_img_tag_srcset( ?Closure $set_up, ?array $custom_sizes ): void {
```

Comment:
See my other comment, I think you can remove the `array $custom_sizes` parameter.